pkg/kubelet/nodeshutdown/nodeshutdown_manager_linux.go

```diff
@@ -73,20 +66,16 @@ type managerImpl struct {
 	nodeRef      *v1.ObjectReference
 	probeManager prober.Manager
 
-	shutdownGracePeriodByPodPriority []kubeletconfig.ShutdownGracePeriodByPodPriority
-
 	getPods        eviction.ActivePodsFunc
-	killPodFunc    eviction.KillPodFunc
 	syncNodeStatus func()
+	podHandler     podHandler
```

Comment:
nit: rename to "podManager" seems better sync with other part naming?

Reply:
Updated